scripts/split_codes.rb

```diff
@@ -19,7 +19,10 @@ def split(input, output)
     end
 
     def swap_term_refs(string)
-      string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>,\1}}')
+      # string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>,\1}}')
+      string.gsub(/\{\{\s*([^}]+)\s*,\s*([^}]+)\s*\}\}/, '{{<<\2>>}}')
+            .gsub(/\(\{\{([^}]+)\}\}\}/, '{{\1}}')
```

Comment:
There's an issue with the replacement, the brackets are not matching.

Reply:
Fixed